addons/account/models/chart_template.py

```diff
@@ -901,11 +901,8 @@ def execute(self):
         # Install all the templates objects and generate the real objects
         acc_template_ref, taxes_ref = self.chart_template_id._install_template(company, code_digits=self.code_digits, transfer_account_id=self.transfer_account_id)
 
-        # write values of default taxes for product as super user
-        if self.sale_tax_id and taxes_ref:
-            ir_values_obj.sudo().set_default('product.template', "taxes_id", [taxes_ref[self.sale_tax_id.id]], for_all_users=True, company_id=company.id)
-        if self.purchase_tax_id and taxes_ref:
-            ir_values_obj.sudo().set_default('product.template', "supplier_taxes_id", [taxes_ref[self.purchase_tax_id.id]], for_all_users=True, company_id=company.id)
+        self.company_id.write({'accounts_sale_tax_id': self.sale_tax_id.id,
```

Comment:
Ok, I tried to follow what was done with the code_digits field which is related too and also writed on the company record a few lines above, I will check that